src/content/docs/validator/external-chains/xrpl.mdx

```diff
@@ -199,3 +196,6 @@ For your node to be recognized by the Axelar network, you need to register your
 ampd register-chain-support amplifier xrpl
 ```
 
+## Troubleshooting
```

Comment:
This is not just trouble shooting, it must be done so should be highlighted in the steps closer to the top